identity-service/src/routes/healthCheck.js

```diff
@@ -10,35 +10,51 @@ const axios = require('axios')
 let notifDiscProv = config.get('notificationDiscoveryProvider')
 
 // Defaults used in relay health check endpoint
-const RELAY_HEALTH_ONE_HOUR_AGO_BLOCKS = 120 // 1 block/5sec = 120 blocks/hr
+const RELAY_HEALTH_TEN_MINS_AGO_BLOCKS = 120 // 1 block/5sec = 120 blocks/10 minutes
 const RELAY_HEALTH_MAX_TRANSACTIONS = 100 // max transactions to look into
 const RELAY_HEALTH_MAX_ERRORS = 5 // max acceptable errors for a 200 response
+const RELAY_HEALTH_MIN_NUM_USERS = 3 //min number of users affected to qualify for error
 const RELAY_HEALTH_MAX_BLOCK_RANGE = 200 // max block range allowed from query params
+const RELAY_HEALTH_SENT_VS_ATTEMPTED_THRESHOLD = 0.2
 const RELAY_HEALTH_ACCOUNT = config.get('relayerPublicKey')
 
+// flatten one level of nexted arrays
+const flatten = (arr) => arr.reduce((acc, val) => acc.concat(val), []);
+
 module.exports = function (app) {
   /**
    * Relay health check endpoint. Takes the query params startBlock, endBlock, maxTransactions, and maxErrors.
    * If those query params are not specified, use default values.
    */
+  /*
+  There are a few scenarios where a health check should return unhealthy
+  1. Some number of relays are failing for some number of users
+     To solve this, have an object in redis which 
```

Comment:
is there meant to be more to this comment?

Reply:
good catch, fixed